tracing-undertow/src/main/java/com/palantir/tracing/undertow/TracedOperationHandler.java

```diff
@@ -62,8 +62,11 @@ public TracedOperationHandler(HttpHandler delegate, String operation) {
     @Override
     public void handleRequest(HttpServerExchange exchange) throws Exception {
         String traceId = initializeTrace(exchange);
+        String sampled = Tracer.isTraceObservable() ? "1" : "0";
+
         // Populate response before calling delegate since delegate might commit the response.
         exchange.getResponseHeaders().put(TRACE_ID, traceId);
+        exchange.getResponseHeaders().put(IS_SAMPLED, sampled);
```

Comment:
Can we create an attachment for this instead of using the response headers?

Reply:
Done. Is this how attachments should be used?

Reply:
Yep, thanks!